src/DataDryadAPI.jl

```diff
@@ -0,0 +1,67 @@
+immutable DataDryadAPI <: DataRepo
+end
+
+base_url(::DataDryadAPI) = "https://datadryad.org/mn/object/http://dx.doi.org/"
+
+function description(repo::DataDryadAPI, mainpage)
+    desc = text_only(first(matchall(sel"dcterms\:description", mainpage.root)))
+    authors = matchall(sel"dcterms\:creator", mainpage.root)
+    author = text_only(first(authors))
```

Comment:
rather than doing this directly, each time 
I suggest you create a general function, call it `short_author(authors)`  which takes in the full author list,
and if only 1 returns just that one.
If 2 returns the first and the second
and if 3 + returns first et al.

Maybe using dispatch and splatting is a good way: 
That would look like

```
short_author(author1) = author1
short_author(author1, author2) = author1 * " and " * author2
short_author(author1, authors...) = author1 * " et al."
```

Invoking it as `short_author(authors...)` to splat the argument


Also note it is "et al." , with out the period after the "et"